### ISSUE TITLE
Confusing export error message

### ISSUE THREAD

**Issue by tgerulaitis** [meanbee/magedbm#41]
I've tried to upload a database with the `put` command, but it failed with an error message:

```
> docker-compose run --rm tools magedbm put test
Starting test_appdata_1
Starting test_dbdata_1
[ ok ] Starting enhanced syslogd: rsyslogd.
Successfully configured AWS credentials.
Successfully configured AWS region config.
Successfully configured magedbm config.


  Dump MySQL Database (without exec)


No-data export for: catalogsearch_fulltext catalogsearch_query catalogsearch_result core_session customer_address_entity customer_address_entity_datetime customer_address_entity_decimal customer_address_entity_int customer_address_entity_text customer_address_entity_varchar customer_entity customer_entity_datetime customer_entity_decimal customer_entity_int customer_entity_text customer_entity_varchar dataflow_batch dataflow_batch_export dataflow_batch_import dataflow_import_data dataflow_session importexport_importdata log_url log_url_info log_visitor log_visitor_info log_visitor_online newsletter_problem newsletter_queue newsletter_queue_link newsletter_queue_store_link newsletter_subscriber newsletter_template report_compared_product_index report_event report_viewed_product_aggregated_daily report_viewed_product_aggregated_monthly report_viewed_product_aggregated_yearly report_viewed_product_index sales_bestsellers_aggregated_daily sales_bestsellers_aggregated_monthly sales_bestsellers_aggregated_yearly sales_flat_creditmemo sales_flat_creditmemo_comment sales_flat_creditmemo_grid sales_flat_creditmemo_item sales_flat_invoice sales_flat_invoice_comment sales_flat_invoice_grid sales_flat_invoice_item sales_flat_order sales_flat_order_address sales_flat_order_grid sales_flat_order_item sales_flat_order_payment sales_flat_order_status_history sales_flat_quote sales_flat_quote_address sales_flat_quote_address_item sales_flat_quote_item sales_flat_quote_item_option sales_flat_quote_payment sales_flat_quote_shipping_rate sales_flat_shipment sales_flat_shipment_comment sales_flat_shipment_grid sales_flat_shipment_item sales_flat_shipment_track sales_order_aggregated_created sales_order_aggregated_updated sales_order_tax sales_order_tax_item sales_payment_transaction sales_recurring_profile sales_recurring_profile_order sales_refunded_aggregated sales_refunded_aggregated_order wishlist wishlist_item wishlist_item_option
Start dumping database magento to file /tmp/magedbm/test-2017-01-16_111758.sql.gz



  [Exception]
  Unable to export database.



put [-s|--strip [STRIP]] [--no-clean] [--history-count HISTORY-COUNT] [-r|--region REGION] [-b|--bucket BUCKET] [--] <name>



```

Unfortunately, the error message does not offer any reasons why it failed, making it impossible to figure out whether it was a configuration error, environment error or an actual bug.

**Comment by bobbyshaw** [comment]
The action here is to ensure that the original exception message is printer as well as our own message.

**Comment by bobbyshaw** [comment]
Available in latest release v1.5.0
